haptools/data/genotypes.py

```diff
@@ -11,6 +11,7 @@
 import numpy.typing as npt
 from cyvcf2 import VCF, Variant
 from pysam import VariantFile, TabixFile
+from . import tr_harmonizer as trh
```

Comment:
```suggestion
try:
    import trtools.utils.tr_harmonizer as trh
except ModuleNotFoundError:
    from . import tr_harmonizer as trh
```
might wanna try to import from trtools first just in case there's any new code from there that we should be using